commit 53c7d19df4dad21fa1bb912485eb6dbd9108491c
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-05-17T10:37:57+00:00 (5 months ago)
    
    TOOLS-2178 eng check targets should reference resources in deps/eng/tools

diff --git a/Makefile b/Makefile
index 26cb7b1..7358747 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
@@ -41,6 +41,7 @@ TOP ?= $(error Unable to access eng.git submodule Makefiles.)
 include ./deps/eng/tools/mk/Makefile.node_prebuilt.defs
 include ./deps/eng/tools/mk/Makefile.agent_prebuilt.defs
 include ./deps/eng/tools/mk/Makefile.node_modules.defs
+include ./deps/eng/tools/mk/Makefile.smf.defs
 
 .PHONY: all
 all: $(STAMP_NODE_PREBUILT) $(STAMP_NODE_MODULES)
@@ -70,7 +71,7 @@ BOOT_DIR =			/opt/smartdc/boot
 SAPI_MANIFESTS =		manta-reshard
 SAPI_MANIFEST_DIRS =		$(SAPI_MANIFESTS:%=$(PREFIX)/sapi_manifests/%)
 
-SMF_MANIFESTS =			reshard
+SMF_MANIFESTS =			smf/manifests/reshard.xml
 SMF_MANIFESTS_DIR =		$(PREFIX)/smf/manifests
 
 NODE_BITS =			bin/node \
@@ -83,7 +84,7 @@ INSTALL_FILES =			$(addprefix $(PROTO), \
 				$(BOOT_SCRIPTS:%=$(BOOT_DIR)/%) \
 				$(SCRIPTS:%=$(SCRIPTS_DIR)/%) \
 				$(TEMPLATES:%=$(TEMPLATES_DIR)/%) \
-				$(SMF_MANIFESTS:%=$(SMF_MANIFESTS_DIR)/%.xml) \
+				$(SMF_MANIFESTS:%=$(PREFIX)/%) \
 				$(NODE_BITS:%=$(NODE_DIR)/%) \
 				$(NODE_MODULE_INSTALL) \
 				$(COMMANDS:%=$(PREFIX)/cmd/%.js) \
@@ -184,3 +185,4 @@ include ./deps/eng/tools/mk/Makefile.targ
 include ./deps/eng/tools/mk/Makefile.node_prebuilt.targ
 include ./deps/eng/tools/mk/Makefile.agent_prebuilt.targ
 include ./deps/eng/tools/mk/Makefile.node_modules.targ
+include ./deps/eng/tools/mk/Makefile.smf.targ
diff --git a/deps/eng b/deps/eng
index 126c0f0..d25b8fc 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit 126c0f032b8bfddd45b14e8ee14e73e9798a013f
+Subproject commit d25b8fc60fb8c649559924870fe3aaf75e7421d5
